actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -105,9 +105,15 @@ def redirect_to(options = {}, response_options = {})
       raise AbstractController::DoubleRenderError if response_body
 
       allow_other_host = response_options.delete(:allow_other_host) { _allow_other_host }
-
       self.status = _extract_redirect_to_status(options, response_options)
 
+      unless redirect?
```

Comment:
Is it possible to achieve this behavior without delegating the `#redirect?` predicate?

[ActionController::Metal](https://edgeapi.rubyonrails.org/classes/ActionController/Metal.html) defines a `#response` attribute. Is that the same instance as `@_response`?

```suggestion
      unless response.redirect?
```

Reply:
thanks for the input!

I originally started with what you've suggested; it works 👍 

Hm it probably does make sense to keep from widening `ActionController::Metal` surface area; upon closer inspection there's nothing else in the API that looks "convenience-y" like `redirect?`

happy to make that change